src/CallHandler.js

```diff
@@ -430,7 +430,7 @@ async function _startCallApp(roomId, type) {
         return;
     }
 
-    const confId = `JitsiConference_${generateHumanReadableId()}`;
+    const confId = `JitsiConference${generateHumanReadableId()}`;
```

Comment:
While we're here... Is there any value in this "JitsiConference" prefix? It seems a bit redundant to me.

Reply:
It's here because Jitsi shows the title, and we wanted to show "Jitsi Conference: Riot Web Meeting" or similar. Someone from the product team could probably clarify how important this is.

Reply:
Ah okay, well we can debate that value of that separately from this PR then. Thanks for the context!